packages/internal/src/ts2js.ts

```diff
@@ -38,10 +38,10 @@ export const convertTsFilesToJs = (cwd: string, files: string[]) => {
     console.log('No TypeScript files found to convert to JS in this project.')
   }
   for (const f of files) {
-    const code = transformTSToJS(f)
+    const code = transformTSToJS(cwd, f)
```

Comment:
Left a comment on the thread, I think it might be better to execute the function with execa, rather than change the function signature here

Reply:
Yep agreed, that thing is used in other places, changing the signature for everyone would not be good! `execa` version works, I created the test project and everything still converted to `.jsx` as expected!